src/query/block/column.go

```diff
@@ -36,6 +36,11 @@ type columnBlock struct {
 	seriesMeta []SeriesMeta
 }
 
+// Unconsolidated returns the unconsolidated version for the block
```

Comment:
nit: `of the block`

Reply:
we use for the block at most places. Can do a search and replace in a separate diff